packages/amazonq/src/extensionNode.ts

```diff
@@ -51,6 +52,9 @@ async function activateAmazonQNode(context: vscode.ExtensionContext) {
         registerSubmitFeedback(context, 'Amazon Q', amazonQContextPrefix)
     )
 
+    globals.schemaService = new SchemaService()
+    filetypes.activate()
```

Comment:
this looks like another case that was missed because we don't yet have a startup "base" that is used by both Q and Toolkit : https://taskei.amazon.dev/tasks/IDE-14189